metau-capstone/StudyViewController.m

```diff
@@ -26,9 +26,11 @@ @implementation StudyViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
+    PFUser *user = [PFUser currentUser];
     // Construct Query
     PFQuery *query = [PFQuery queryWithClassName:@"Flashcard"];
+    [query whereKey:@"userID" equalTo:user.objectId];
+    [query whereKey:@"levelNum" equalTo:@(1)];
```

Comment:
Nice!

Reply:
This is likely where some of your complexity will blow up as we figure out what cards to display

Reply:
+1